content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -395,3 +395,71 @@ See the following resources:
 - [Transaction isolation levels in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
 
 CockroachDB and SQLite only support the `Serializable` isolation level.
+
+### Transaction timing issues
+
+<Admonition type="info">
+
+- The solution in this section does not apply to MongoDB, because MongoDB does not support [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
+- The timing issues discussed in this section do not apply to CockroachDB and SQLite, because these databases only support the highest `Serializable` isolation level.
+
+</Admonition>
+
+When two or more transactions run concurrently in certain [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels), timing issues can cause conflicts, such as the violation of unique constraints. For example, consider the following sequence of events where Transaction A and Tranasaction B both attempt to execute a `deleteMany` and a `createMany` operation:
+
+1. Transaction A: `deleteMany` operation removes a set of rows.
+1. Transaction B: `deleteMany` operation removes a set of rows.
+1. Transaction B: `createMany` operation creates a new set of rows.
+1. Transaction B: The application commits transaction B.
+1. Transaction A: `createMany` operation. The newly-inserted rows conflict with the rows that transaction B added at step 4.
+
+This conflict can occur at the isolation level `ReadCommited`, which is the default isolation level in PostgreSQL and Microsoft SQL Server. To avoid this problem, you can set a higher isolation level (`RepeatableRead` or `Serializable`).
+
+From Prisma version 4.4.0, you do not need to set the isolation level in your database configuration. You can instead set the isolation level on a transaction. This overrides the isolation level on your database for that transaction. [Learn more about isolation levels in Prisma tranasctions](#transaction-isolation-level).
```

Comment:
I think we could turn this around, and after the explanation of how to set it in the transaction via `isolationLevel` mention that up to Prisma 4.4.0 this was not possible and you had to configure this in your database directly.

Reply:
I've reworked this section to reflect your suggestion.